Add FAQ entry about first blocks on Alephium blockchain

sven-hash opened the pull request:
Added information about the first blocks on the Alephium blockchain, including the genesis and second blocks with links to the explorer.

sven-hash committed:
Add FAQ entry about first blocks on Alephium blockchain

Added information about the first blocks on the Alephium blockchain, including the genesis and second blocks with links to the explorer.

polarker commented on docs/misc/FAQ.md:
```diff
@@ -597,6 +597,15 @@ val LaunchTimestamp: TimeStamp  = TimeStamp.unsafe(1636379973000L) // 2021-11-08
+The Alephium blockchain began with the genesis block:
+
+- **Genesis Block:** [`9de358fd188ce5c2ec7fef45e00974435da7e22992653a7ee828a441634cb950`](https://explorer.alephium.org/blocks/9de358fd188ce5c2ec7fef45e00974435da7e22992653a7ee828a441634cb950)
+- **Second Block:** [`0000000003192d50fff87253215a18b48031e94e47b25de98b62a5eba5b90780`](https://explorer.alephium.org/blocks/0000000003192d50fff87253215a18b48031e94e47b25de98b62a5eba5b90780)
```
The first few blocks should be parallel, with the same timestamp.

sven-hash replied:
Right so we should have 4 genesis block and 4 seconds blocks?

polarker replied:
We should have 16 genesis blocks, the rests are mined by miners

sven-hash committed:
Revise FAQ to include block tables

Updated the FAQ section to include a table format for the genesis and second blocks on the Alephium blockchain, improving readability and organization.
